src/registry_schemas/schemas/amalgamation_application.json

```diff
@@ -92,18 +87,49 @@
                         "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/parties#/definitions/party"
                     }
                 },
-                "shareStructure": {
-                    "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/share_structure"
-                },
                 "courtOrder": {
                     "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/court_order#/properties/courtOrder"
                 },
                 "courtApproval": {
                     "title": "Amalgamation statement",
                     "type": "boolean"
                 }
-            }
-
+            },
+            "allOf": [
+                {
+                    "if": {
+                        "properties": {
+                            "type": {
+                                "enum": [
+                                    "regular"
```

Comment:
Do we care?
Is there an issue if they change these?

Reply:
Yes. As per the requirement office, shares and director should be copied from Holding/Primary Company. So it should not be modified